.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+# Elixir CircleCI 2.0 configuration file
```

Comment:
We have had _really_ bad experiences with Circle CI 2.0 (compared to 1.0) There is a lot of upheaval in their community about it missing features and being unstable. 

That said, this config is pretty simple. If it works, it works. Just keep that in mind if we start having trouble and don't be afraid to switch to Circle CI 1.0 or another platform.